samples/99_zil_interpreter/tests/parser_tests_basic.rb

```diff
@@ -466,6 +466,123 @@ def test_parser_24(args, assert)
   assert.equal! parsed, expected
 end
 
+# test unmatched <
+# --------------------------------------------------------------------
+def test_parser_unmatched_1(args, assert)
+  source = "<+ 2 3 4"
+
+  begin
+    Parser.parse_string(source)[0]
+    raise 'Parser exception expected!'
+  rescue
+    assert.ok!
+  end
```

Comment:
Maybe extracting a helper might make this a bit shorter

maybe like this? ;)
```rb
assert_raises_parser_error!(assert, source)
```

Also you probably don't need the `[0]` behind the method call

Reply:
Good idea.  I've added this into current version.